Source/trigs.cpp

```diff
@@ -58,6 +58,34 @@ int L6TWarpUpList[] = { 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92,
 int L6UpList[] = { 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, -1 };
 int L6DownList[] = { 57, 58, 59, 60, 61, 62, 63, 64, -1 };
 
+const char *LvlNames[] = {
+	"Tristram",
+	"Cathedral 1",
+	"Cathedral 2",
+	"Cathedral 3",
+	"Cathedral 4",
+	"Catacombs 1",
+	"Catacombs 2",
+	"Catacombs 3",
+	"Catacombs 4",
+	"Caves 1",
+	"Caves 2",
+	"Caves 3",
+	"Caves 4",
+	"Hell 1",
+	"Hell 2",
+	"Hell 3",
+	"Diablo",
+	"Nest 1",
+	"Nest 2",
+	"Nest 3",
+	"Nest 4",
+	"Crypt 1",
+	"Crypt 2",
+	"Crypt 3",
+	"Crypt 4",
+};

```

Comment:
Starting the count back in 1 for each tileset is a pretty big change in design/purpose. The old number was supposed to represent overall depth and this is completely lost with this change. The restarting number makes each tileset "disconnected" with the others, it loses the "sequence" aspect. That could be fine for hive and crypt, but it really goes against the original non-Hellfire design in Diablo.

Additionally, I'm really not a fan of having a hardcoded array of names like this that includes the number.. I'd rather have a function that generates the name based on the depth, and tileset names.